src/VirtoCommerce.Platform.Web/wwwroot/js/common/uiGridUtils.js

```diff
@@ -165,6 +191,14 @@ angular.module('platformWebApp')
                 if ($scope && $scope.gridApi)
                     $scope.gridApi.core.notifyDataChange(uiGridConstants.dataChange.COLUMN);
             }
+		
+            function toSentenceCase(str)
+            {
+                 // adding space between strings
+                const result = str.replace(/([A-Z])/g,' $1');
```

Comment:
I think you need to Trim() the first space.
`NiceString` -> `<SPACE>Nice<SPACE>String`